.eslintrc

```diff
@@ -27,7 +27,10 @@
     "import/no-extraneous-dependencies": 0,
     "import/newline-after-import": 0,
     "no-mixed-operators": 0,
-    "quotes": ["error", "double"],
-    "arrow-parens": ["error", "as-needed"]
+    "quotes": ["warn", "double"],
```

Comment:
Relaxing the quotes rule as we will eventually change this to `single` to align with HubSpot's style.

Reply:
Could you create a ticket for us to update the quotes to single and update this rule?

Reply:
https://issues.hubspotcentral.com/browse/CALL-3282